test_conformance/extensions/cl_khr_command_buffer/main.cpp

```diff
@@ -63,7 +63,13 @@ test_definition test_list[] = {
     ADD_TEST(event_info_context),
     ADD_TEST(event_info_reference_count),
     ADD_TEST(finalize_invalid),
-    ADD_TEST(finalize_empty)
+    ADD_TEST(finalize_empty),
+    ADD_TEST(create_command_buffer_num_queues),
+    ADD_TEST(create_command_buffer_null_queues),
+    ADD_TEST(create_command_buffer_repeated_properties),
+    ADD_TEST(create_command_buffer_not_supported_properties),
+    ADD_TEST(create_command_buffer_queue_without_min_properties),
+    ADD_TEST(create_command_buffer_device_does_not_support_out_of_order_queue),
```

Comment:
Can we prefix the names of the new tests with `negative_` here and going forward, that way a user can filter to run only negative tests by invoking `$ ./test_conformance/extensions/cl_khr_command_buffer/test_cl_khr_command_buffer negative*`